subcommand/consul-sidecar/command_test.go

```diff
@@ -31,13 +31,13 @@ func TestRun_Defaults(t *testing.T) {
 }
 
 func TestRun_ExitsCleanlyonSignals(t *testing.T) {
```

Comment:
Is there a reason we have these test cases using this pattern of a top-level function that calls a private method rather than using the typical pattern of cases declared and then looping over those cases:

```go
func TestRun_SignalHandlingRegistration(t *testing.T) {
	cases := map[string]os.Signal{
		"SIGINT":  syscall.SIGINT,
		"SIGTERM": syscall.SIGTERM,
	}

	for name, signal := range cases {
		t.Run(name, func(t *testing.T) {

```

Reply:
I think we should use the typical pattern, I went ahead and switched that. Thanks!